### PR TITLE
Filter the device and update alerts to only the ones that are Active

### PR DESCRIPTION
**Pull request by adeolau-ms**
## PR Checklist

* [x] Filter the device and update alerts to just the ones that are active.

### CODE REVIEW

File: Workbooks/UpdateCompliance/Feature updates/Feature updates.workbook [hunk 1]
```diff
@@ -86,8 +86,7 @@
             ],
             "isHiddenWhenLocked": true,
             "queryType": 0,
-            "resourceType": "microsoft.operationalinsights/workspaces",
-            "value": ""
```
Comment (mtilak) [inline Workbooks/UpdateCompliance/Feature updates/Feature updates.workbook:90]: >"value": "" [](http://example.com/codeflow?start=12&length=11)

Removing this intentionally?

**Review by mtilak — APPROVED** [review]
:shipit:

File: Workbooks/UpdateCompliance/Alerts information/Device Alerts.workbook [hunk 2]
```diff
@@ -233,7 +234,7 @@
                       {
                         "operator": "Default",
                         "thresholdValue": null,
-                        "representation": "Alert",
+                        "representation": "error",
```
Comment (ranjith513) [inline Workbooks/UpdateCompliance/Alerts information/Device Alerts.workbook:237]: significance of this change..?

File: Workbooks/UpdateCompliance/Feature updates/Feature updates.workbook (hunk 1 above)
Comment (adeolau-ms) [reply Workbooks/UpdateCompliance/Feature updates/Feature updates.workbook]: Fixed

File: Workbooks/UpdateCompliance/Alerts information/Device Alerts.workbook (hunk 2 above)
Comment (adeolau-ms) [reply Workbooks/UpdateCompliance/Alerts information/Device Alerts.workbook]: Fixed

**Review by ranjith513 — APPROVED** [review]

**Pull request merged**